src/LogExpert/Extensions/ToolStripRendererExtension.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace LogExpert.Extensions
+{
+    public class ToolStripRendererExtension : ToolStripSystemRenderer
+    {
+        public ToolStripRendererExtension() { }
+
+        protected override void OnRenderToolStripBorder(ToolStripRenderEventArgs e)
+        {
+            //base.OnRenderToolStripBorder(e);
```

Comment:
is here the implementation missing? 😕

Reply:
I added a comment to be more clear, it is on purpose to not draw a border visible in Dark Mode